playgame.py

```diff
@@ -25,7 +25,7 @@ def __init__(self, message, player_id):
         self.player_id = player_id
 
 def main(argv):
-    args = parse_args()
+    args = parse_args(argv)
```

Comment:
I like passing in the array explicitly here.

Reply:
yea, it led to silly hidden errors...